prow/scripts/lib/log.sh

```diff
@@ -11,34 +11,46 @@ function log::date {
     date +"%Y/%m/%d %T %Z"
 }
 
+# log::banner prints message with INFO level in banner form for easier spotting in log files
+#
+# Arguments:
+#   $1 - Message
```

Comment:
The same situation as here: https://github.com/kyma-project/test-infra/pull/2708/files#r494070514